gitpod-io/gitpod
# workspackit: put the workspace id into logs to trace.

## Description

We can't trace the workspace kit logs because it doesn't have the workspace id in the logs. This PR put a workspace id into the workspacekit log.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/9247

## How to test
[example logs](https://cloudlogging.app.goo.gl/EjKhF94qHoei1rXW7)

1. Open the worksapce
2. Run `sudo mount -t proc proc /proc/11111/`
3. Confirm that the log is included `worksapceId`

## Release Notes

```release-note
workspackit: put the workspace id into logs to trace.
```

## Documentation

No
